WordPress/src/main/java/org/wordpress/android/ui/plugins/PluginUtils.java

```diff
@@ -46,4 +54,27 @@ static boolean isUpdateAvailable(@Nullable ImmutablePluginModel immutablePlugin)
             return !installedVersionStr.equalsIgnoreCase(availableVersionStr);
         }
     }
+
+    public static boolean isJetpack(@NonNull ImmutablePluginModel plugin) {
+        return checkNameOfPlugin(plugin, "jetpack/jetpack");
+    }
+
+    public static boolean isAutoManaged(@NonNull SiteModel site, @NonNull ImmutablePluginModel plugin) {
+        if (!site.isAutomatedTransfer()) {
+            return false;
+        }
+        if (!plugin.isInstalled()) {
+            return false;
+        }
+        boolean isAutoManaged = false;
+        for (String pluginName : AUTO_MANAGED_PLUGINS) {
+            isAutoManaged = isAutoManaged || checkNameOfPlugin(plugin, pluginName);
+        }
+        return isAutoManaged;
+    }
+
+    private static boolean checkNameOfPlugin(@NonNull ImmutablePluginModel plugin, @NonNull String name) {
```

Comment:
Is this method necessary? Seems like `StringUtils.equals(plugin.getName(), name)` would suffice, but if we do keep it a better name would be helpful (`isSamePluginName` perhaps?).